staging/src/k8s.io/client-go/plugin/pkg/client/auth/exec/metrics.go

```diff
@@ -105,6 +105,7 @@ func incrementCallsMetric(err error) {
 		metrics.ExecPluginCalls.Increment(failureExitCode, pluginNotFoundError)
 
 	default: // We don't know about this error type.
+		//nolint:logcheck // This looks like debug output which shouldn't occur in practice.
 		klog.V(2).InfoS("unexpected exec plugin return error type", "type", reflect.TypeOf(err).String(), "err", err)
```

Comment:
Shouldn't we update this instead of ignoring?

Reply:
We can update to structured logging, but doing contextual logging didn't seem worth the chain of API changes.

Reply:
Wait, it is already using structured logging. The suppression is for the "contextual logging" aspect.

I can rewrite as `klog.Background().V(2).Info` instead of suppressing the linter, but proper conversion (= avoiding the global logger) probably isn't worth it.